opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -153,6 +153,12 @@ namespace Opm {
         const std::map<int, VFPInjTable>& getVFPInjTables() const;
         size_t getNumPhases() const;
 
+        // The saturation function family.
+        // If SWOF and SGOF are specified in the deck it return 1
+        // If SWFN, SGFN and SOF3 are specified in the deck it return 2
+        // If keywords are missing or mixed, an error is given.
+        const size_t getSaturationFunctionFamily() const;
```

Comment:
>it is needed in the opm-material code as well. I agree with the enum, but I can't really come up with good names.

As long as there are only two families I guess it could be a `bool`?